src/battle_script_commands.c

```diff
@@ -8444,6 +8446,9 @@ static bool32 CourtChangeSwapSideStatuses(void)
     UPDATE_COURTCHANGED_BATTLER(tailwindBattlerId);
     UPDATE_COURTCHANGED_BATTLER(luckyChantBattlerId);
     UPDATE_COURTCHANGED_BATTLER(stickyWebBattlerId);
+    UPDATE_COURTCHANGED_BATTLER(rainbowBattlerId);
+    UPDATE_COURTCHANGED_BATTLER(seaOfFireBattlerId);
+    UPDATE_COURTCHANGED_BATTLER(swampBattlerId);
```

Comment:
You don't need battler ids for these effects as they persist irrespective of the Pokemon that used them. You can swap the timers with `COURTCHANGE_SWAP`.

Reply:
I can't use `COURTCHANGE_SWAP` since there is no status flag. Would using `SWAP` be enough?

Reply:
Might be good to add a side status flag then, as these are side statuses after all.

Reply:
Added a status flag